Terminal.Gui/View/ViewDrawing.cs

```diff
@@ -236,6 +236,16 @@ public void Draw ()
 
         OnRenderLineCanvas ();
 
+        // TODO: This is a hack to force the border subviews to draw.
+        if (Border?.Subviews is { })
```

Comment:
This made me go look at the TreeView stuff.

I now have more things on my list to bring up later. 😄